upgrade/rendering-changes/6-3-0.md

```diff
@@ -0,0 +1,64 @@
+---
+title: 6.3.0
```

Comment:
A side note - ideally, the rendering changes should have been in a separate PR.

Reply:
Indeed, this is the practice. I had to add such an article here anyway to list the changes in the PDFViewer Toolbar. I also plugged the rest of the rendering changes here to save some time.